### PR TITLE
docs: v3 installation + errors

### PR DESCRIPTION
**Pull request by justchokingaround**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [ ] Feature
- [X] Documentation update

## Description

*ramble here*

## Checklist

- [ ] any anime playing
- [ ] bumped version
- [ ] next, prev and replay work
- [ ] quality works
- [ ] downloads work
- [ ] quality works with downloads
- [ ] select episode -a and rapid resume work
- [ ] syncplay -s works
- [ ] autoplay, aka range selection, works

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Not uploaded: one piece dub episode 590
- Unreleased: soredemo ayumu wa yosetekuru
- Short id (for decryption): Log Horizon episode 1-2

### CODE REVIEW

**Review by port19x — APPROVED** [review]

**Comment by port19x** [comment]
Resolved the merge conflict

**Comment by DaBigBlob** [comment]
@port19x @justchokingaround 
ayo what was wrong with this: https://github.com/pystardust/ani-cli/pull/756

**Comment by port19x** [comment]
> @port19x @justchokingaround ayo what was wrong with this: #756

not 100% sure, but since choker made a branch I'll prefer it over your PR

File: README.md [hunk 1]
```diff
@@ -72,27 +81,27 @@ yay -S ani-cli
...
 
 Install dependencies [(See below)](#Dependencies)
 
 ```sh
-sudo curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o /usr/local/bin/ani-cli &&
-sudo chmod +x /usr/local/bin/ani-cli
+git clone https://github.com/pystardust/ani-cli.git && cd ani-cli
+chmod +x ani-cli && sudo cp ani-cli /usr/local/bin/ani-cli
+chmod +x UI && sudo cp UI /usr/local/bin/UI
+sudo cp -r players /usr/local/bin/players
```
Comment (DaBigBlob) [inline README.md:92]: apple silicon macs use a different installation path for brew than intel ones
and if the user already has brew then $(brew --prefix) is already safe and is in path

File: README.md [hunk 2]
```diff
@@ -72,27 +81,27 @@ yay -S ani-cli
...
-Install dependencies [(See below)](#Dependencies)
+<br>
 
-Install [HomeBrew](https://docs.brew.sh/Installation) if not installed.
+*To install (with [HomeBrew](https://docs.brew.sh/Installation)) the dependencies required on Mac OS, you can run:*
 
 ```sh
-curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o "$(brew --prefix)/bin/ani-cli"
-chmod +x "$(brew --prefix)/bin/ani-cli"
-```
```
Comment (DaBigBlob) [inline README.md:95]: dont reamove this [previous comment]

File: README.md (hunk 1 above)
Comment (DaBigBlob) [reply README.md:92]: Screen Shot 2022-06-15 at 1 16 54 PM

**Comment by tpeacock19** [comment]
I would suggest taking direction from @DaBigBlob's PR and changing:

```sh
project_root="$(dirname "$0")"
```
to either

```sh
project_root="$(dirname "$0" | sed 's/....$//')/share/ani-cli"
```
or
```sh
project_root="$(cd "$(dirname "$BASH_SOURCE[0]")/../" && pwd)/share/ani-cli"
```

I dont think its appropriate to place non-executables nor create directories in a `bin` folder.

**Comment by justchokingaround** [comment]
closing in favor of #762

**Comment by tpeacock19** [comment]
>$BASH_SOURCE might be a bad idea
>not everyone uses bash


if that is the case then using this would suffice and is clearer than the sed solution.

```
project_root="$(cd "$(dirname $0)/../" && pwd)/share/ani-cli"
```

**Pull request closed without merging**